components/resources/wallet_strings.grdp

```diff
@@ -1027,4 +1027,7 @@
   <message name="IDS_BRAVE_WALLET_TOTAL_BALANCE_DESCRIPTION" desc="Total balance tooltip description">Your available funds plus any not-yet-confirmed transactions.</message>
   <message name="IDS_BRAVE_WALLET_UNAVAILABLE_BALANCES" desc="Some balances may be unavailable info bar">Some balances may be unavailable</message>
   <message name="IDS_BRAVE_WALLET_FOUND_RISKS" desc="Notice that a number of risks were found">We found <ph name="VALUE">$1<ex>2</ex></ph> risks.</message>
+  <message name="IDS_BRAVE_WALLET_RISK_OF_LOSS_ACTION" desc="Warning that the pending action may result in loss of funds">Unsafe action with risk of loss</message>
+  <message name="IDS_BRAVE_WALLET_UNTRUSTED_REQUEST_WARNING" desc="Warning that the pending request may be malicious">This request appears unstable or malicious, and may leave you vulnerable to theft. Do not continue unless you trust the app.</message>
```

Comment:
nit: do we need a period at the end? Not sure actually if we have any consistent ux guidance on this.

Reply:
My guess as to why this does not have a period is because it is not a complete sentence